src/main/scala/cromwell/binding/package.scala

```diff
@@ -32,6 +33,9 @@ package object binding {
    */
   implicit class EnhancedRuntimeAttributes(val runtimeAttributes: RuntimeAttributes) extends AnyVal {
     def docker: Option[String] = attribute("docker")
+    def failOnStderr: Boolean =
+      attribute(RuntimeAttribute.failOnStderr.toString).isDefined &&
+        attribute(RuntimeAttribute.failOnStderr.toString).get.equalsIgnoreCase("true")
```

Comment:
This is where I share my link to a blog post w/ the awesome `Option` cheat sheet: http://blog.originate.com/blog/2014/06/15/idiomatic-scala-your-options-do-not-match/